src/html-dynamic-cdn-webpack-plugin.js

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line no-unused-vars
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import HtmlWebpackIncludeAssetsPlugin from 'html-webpack-include-assets-plugin';
+
+import DynamicCdnWebpackPlugin, {pluginName} from './dynamic-cdn-webpack-plugin';
+
+export default class HtmlDynamicCdnWebpackPlugin extends DynamicCdnWebpackPlugin {
```

Comment:
Nit: I feel there is no need to export this as `default`. We are renaming the exports in `src/index.js`, so might as well do a named export